.github/workflows/kustomize-build-ci.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: kustomize-build-ci
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'kustomize/**'
+      - '.github/workflows/kustomize-build-ci.yml'
+  pull_request:
+    paths:
+      - 'kustomize/**'
+      - '.github/workflows/kustomize-build-ci.yml'
+jobs:
+  job:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: kustomize build base
+        run: |
+          cd kustomize/
+          kubectl kustomize .
+      - name: kustomize build cymbal-branding
+        run: |
+          cd kustomize/
+          kustomize edit add component components/cymbal-branding
+          kubectl kustomize .
+      - name: kustomize build google-cloud-operations
+        run: |
+          cd kustomize/
+          kustomize edit add component components/google-cloud-operations
+          kubectl kustomize .
+      - name: kustomize build memorystore
+        run: |
+          cd kustomize/
+          kustomize edit add component components/memorystore
+          kubectl kustomize .
+      - name: kustomize build network-policy
+        run: |
+          cd kustomize/
+          kustomize edit add component components/network-policy
```

Comment:
Wondering if we could edit the `kustomize/README.md` to use this approach instead of `nano`, what do you think?

Reply:
Good thought!

The reason I don't want to use `kustomize edit` in Online Boutique documentation is because I don't want Online Boutique users to have to download the `kustomize` command line tool.
One of the main reasons we chose `kustomize` is because it's built into `kubectl` (see [kpt vs kustomize Google Doc here](https://docs.google.com/document/d/1g1ccJz2O0TDHTeoH-uV7_NhAdSBVq-zcvy5RLdvx5j8/edit#bookmark=id.lhdrit54dd9r)).

Let me know if you feel strongly that we should use `kustomize edit`.

Reply:
My thought is that if someone wants to use Kustomize, they will use the tools they will prefer, and I think the `kustomize` CLI offers a good experience rather than `sed`, `nano`, etc. :)

Reply:
Like discussed offline, let's have both `sed` and `kustomize edit` to take into account different CUJs.